compose/cli/main.py

```diff
@@ -267,6 +267,7 @@ def build(self, options):
             --build-arg key=val     Set build-time variables for services.
             --parallel              Build images in parallel.
             -q, --quiet             Don't print anything to STDOUT
+            --exec                  Run docker build command
```

Comment:
could you keep this sorted alphabetically?

Reply:
currently, this codebase is not sorted alphabetically.
should I do it?

Reply:
Yep.

Sorted > Unsorted :smiley: